.github/workflows/ci.yaml

```diff
@@ -7,10 +7,13 @@
 name: CI
 
 # This workflow will run when developer push a topic branch to their
-# fork in github, minimizing noise for maintainers.
+# fork in github, minimizing noise for maintainers;
+# It also runs on nightly basis at 12:00 AM
```

Comment:
Do you know which timezone? if so it would be nice to add the timezone in the comment.